tapeagents/core.py

```diff
@@ -13,10 +13,21 @@
 class TrainingText(BaseModel):
     """
     Data sample to finetune a language model
+    A training sample for finetuning a language model. Contains:
+    - text: The full text sequence (prompt + completion)
+    - n_predicted: Number of tokens in the completion
+    - reward: Reward value for the completion
```

Comment:
can we use "output" instead of "completion" to adhere to `LLM` interface in TapeAgents? 

instead of "assistant model" I'd use "current model"

and maybe can we separate the fields that are necessary for RL from other fields

Reply:
also I'd suggest `parent_tape_id` -> `tape_id`, the shorter name is sufficiently descriptive here